controllers/managedfusionoffering_controller.go

```diff
@@ -45,6 +45,8 @@ const (
 	subscriptionName  = "managed-fusion-offering"
 )
 
+var doesCRDExistsAtStart bool
```

Comment:
We should not use global variables as they are not tied to any reconciliation context (as multipole can run in parallel) and on top of that we should not do cross package global variables (the offering code needs to move to the offering package, this is an open tech debt and we should consider that when adding new code)